guides/common/attributes.adoc

```diff
@@ -194,7 +210,8 @@ ifeval::["{build}" == "satellite"]
 :PlanningDocURL: {BaseURL}planning_for_red_hat_satellite/index#
 :ProvisioningDocURL: {BaseURL}provisioning_guide/index#
 endif::[]
-ifeval::["{build}" == "foreman-deb"]
+
+ifdef::foreman-deb[]
```

Comment:
Ah, we need to avoid spaces in this file:

```
// NOTE: Avoid empty lines so it can be included in the header section
// Document state: "nightly" for master, "stable" for last two releases,
// "unsupported" for the rest and "satellite" for satellite build
```

I haven't found that it breaks anything, but according to docs everything until an empty line is considered to be a "header". Shall we fix this or just ignore this? I am a bit confused by asciidoc here.

Reply:
I checked that build, and it was working. Weird. Let's fix.